src/components/UserProjectsPage/index.jsx

```diff
@@ -43,17 +43,12 @@ class UserProjectsPage extends React.Component {
     getClustersList();
   }
 
+  componentDidUpdate(prevProps) {
+    const { isAdded, data, getUserProjects } = this.props;
 
-  handleChange(e) {
-    const { error } = this.state;
-    this.setState({
-      [e.target.name]: e.target.value
-    });
-
-    if (error) {
-      this.setState({
-        error: ''
-      });
+    if (isAdded !== prevProps.isAdded) {
```

Comment:
@Mubangizi This component reloads the page so fast that the user doesn't get a chance to see the success creation message, so i was requesting you add a one-second delay to cater for the success message display. somethig like
`  setTimeout(
      () => {
        if (isAdded !== prevProps.isAdded) {
          getUserProjects(data.id);
          this.hideForm();
        }
      }, 1000
    );
  }
`

Reply:
@MutegekiHenry. I think it's okay without the success message.
If you remember from the conversation we had on one of the calls, we said setTimeout could introduce bugs. Therefore, it is better we get rid of them. In this case, the component re-rendering with the new project is sufficient indication of success.

Reply:
yeah me too, i find it more convenient when you see the component, load and go off, telling you that you request was successful